src/Esprima/Utils/AstJson.cs

```diff
@@ -168,30 +168,30 @@ void WriteLocationInfo(Node node)
                 {
                     if (includeRange)
                     {
-                        writer.Member("range");
```

Comment:
compiler issued a warning as the member field is checked against null but the parameter is not (these use parameter event though member has been initialized)